index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/brands.min.css" integrity="sha512-lCU0XyQA8yobR7ychVxEOU5rcxs0+aYh/9gNDLaybsgW9hdrtqczjfKVNIS5doY0Y5627/+3UVuoGv7p8QsUFw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style/style.css">
+    <title>Portifolio - Figma layout</title>
+    
+</head>
+<body>
+    <header>
+        <nav>
+          
+            <a class="logo" href="#">Willian.T</a>
+            <a href="#"><img class="button-mobile" src="../img/mobile-button.png" alt="mobile-icon"></a>
+        </nav>
+        <section class="headline">
+            <h1>I’m Willian Teixeira Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="">LET'S CONNECT</a>
+            <span class="brands"><i class="fab fa-twitter"></i><i class="fab fa-linkedin-in"></i><i class="fab fa-github"></i><i class="fab fa-angellist"></i></span>
+        </section>
+    </header>
```

Comment:
- [ ] The header and the headline section are two different parts of the body. The header only contains the nav. Kindly take out the headline section from the header and place it after it.
- [ ] Kindly fix the menu icon. It's not showing on the webpage.
![image](https://user-images.githubusercontent.com/43698511/138180536-581e7e04-5ea4-4136-9a9d-70d8d5a48e8f.png)

Reply:
Done, header and headline now are in two different sections.
Menu icon fixed.